lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -334,6 +354,9 @@ private function performOperation($operation)
     private function processReplaceOperation($operation)
     {
         $this->processDeleteOperation( $operation, "diffmod" );
+        if ($this->insertSpaceInReplace) {
+            $this->content .= ' ';
+        }
```

Comment:
Yeah - this is in processReplaceOperation, which means there is always going to be delete and insert. The others are standalone insert and delete operations and so they will be next to a match or a space (equal), so this is the only place that we'll be needing the space inserted, I believe.